packages/cubejs-client-core/index.d.ts

```diff
@@ -252,9 +252,13 @@ declare module '@cubejs-client/core' {
      */
     y?: string[];
     /**
-     * If `true` missing dates on the time dimensions will be filled with `0` for all measures.Note: the `fillMissingDates` option set to `true` will override any **order** applied to the query
+     * If `true` missing dates on the time dimensions will be filled with fillWithValue or `0` by default for all measures.Note: the `fillMissingDates` option set to `true` will override any **order** applied to the query
      */
     fillMissingDates?: boolean | null;
+    /**
+     * Value to autofill all the missing date's measure.
+     */
+    fillWithValue?: string | null;
```

Comment:
Maybe it should be Number instead?

Reply:
And even in tests you use `5` as number, and not as string.

Reply:
@KSDaemon It was meant to be a string. For example lets take an example of users need the auto filled values to be '-' or 'N/A' or 'Missing'. So this enables a scope of allowing users to fill in the missing measure with any value: Number / String.

Reply:
adding an extra testcase of string example